docs/core/install/linux-ubuntu.md

```diff
@@ -44,6 +45,12 @@ The following versions of .NET Core are no longer supported. The downloads for t
 
 [!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
 
+## 20.10 ✔️
+
+.NET 5 and .NET Core 3.1 package feeds for Ubuntu 20.10 currently have an issue. For more information about the issue, see [GitHub](https://github.com/dotnet/core/issues/5549). This article will be updated when the issue is resolved.
```

Comment:
IMO `GitHub` isn't descriptive enough:

```suggestion
.NET 5 and .NET Core 3.1 package feeds for Ubuntu 20.10 currently have an issue. For more information about the issue, see [GitHub issue dotnet/core#5549](https://github.com/dotnet/core/issues/5549). This article will be updated when the issue is resolved.
```

or more abstract:

```suggestion
.NET 5 and .NET Core 3.1 package feeds for Ubuntu 20.10 currently have an issue. For more information about the issue, see [this tracking issue on GitHub](https://github.com/dotnet/core/issues/5549). This article will be updated when the issue is resolved.
```

(Although I don't know what the style guide thinks about `this` in that kind of link.)

Reply:
I prrefer "see [GitHub issue dotnet/core#5549]" "this...issue" is like "click here" which is advised against for accessibility reasons.

Reply:
Hopefully only temporary 😄